cornflow/models/execution.py

```diff
@@ -63,7 +64,11 @@ class ExecutionModel(BaseAttributes):
     execution_results = db.Column(JSON, nullable=True)
     log_text = db.Column(TEXT, nullable=True)
     log_json = db.Column(JSON, nullable=True)
-    finished = db.Column(db.Boolean, server_default=expression.false(), default=False, nullable=False)
+    state = db.Column(db.SmallInteger, default=0, nullable=False) # IntegerRangeField
```

Comment:
it makes sense to have the default at 0? I would import the constants here too and assign the constant with a name. I think it's better to assign something like "UNKNOWN_ERROR" by default.

Reply:
The moment the execution is created, by the default is going to be run, so i guess the default state should be running.

Reply:
yeah but if an error happens somewhere that avoids detection and so an update of the error, then it will show as running for ever. Well, we can leave it like that, yes: it also makes sense.